superchain/superchain.go

```diff
@@ -520,6 +520,11 @@ func init() {
 		}
 		superchainEntry.Superchain = s.Name()
 
+		// HACK: special case for boba-sepolia
+		if s.Name() == "boba-sepolia" {
+			superchainEntry.Superchain = "sepolia"
+		}
```

Comment:
These changes are used to minimize the changes in erigon:
```
if strings.EqualFold(chainCfg.Chain+"-"+chainCfg.Superchain, name) {
	return chainCfg
}
```
If we use `boba-sepolia` as the chain name, the chain variable in erigon will become `boba-sepolia-boba`.